cmd/entrypoint/main.go

```diff
@@ -33,26 +32,35 @@ var (
 	waitFiles       = flag.String("wait_file", "", "Comma-separated list of paths to wait for")
 	waitFileContent = flag.Bool("wait_file_content", false, "If specified, expect wait_file to have content")
 	postFile        = flag.String("post_file", "", "If specified, file to write upon completion")
-
-	waitPollingInterval = time.Second
+	runner          = flag.String("runner", "default", "Valid values are default or logging")
 )
 
 func main() {
 	flag.Parse()
 
+	var r entrypoint.Runner
+	switch *runner {
+	case "logging":
+		r = &entrypoint.LoggingRunner{}
+	case "default":
+		fallthrough
+	default:
```

Comment:
Should this fail instead?

Reply:
Looks like @dlorenc added the comment above this case to explain its behaviour.  @ImJasonH are you happy with this to go in as-is?

Apart from this question I'm ready to lgtm.

Reply:
im still not clear on this but dont really want to leave this open indefinitely - i do feel like not failing when the logging isnt configured correctly probably makes sense (if that is whats happening here?